actions/validate.js

```diff
@@ -22,12 +22,14 @@ function validate(arg) {
   doValidation(path, form, key);
 
   if (Array.isArray(field.dependents) && field.dependents.length) {
+  	
     field.dependents.forEach(function(dependent) {
       var path = joinPath(input.field, dependent);
-      var key = path.pop();
-      var form = state.get(path);
-
-      doValidation(path, form, key);
+      if( path ){
```

Comment:
Yes, @abalmos added this to be able to depend on other forms like formA: {} , formB: {} and be able to validate against other forms or field in the same form. It also updates the field if another one is validated. But there was a very cryptic error when I was using [] and forgot [[]] and it took me a while to see the bug. Therefor I added this so the next programmer didn't have to spend some time on this. But maybe it's better to just let it fail.